oreoluwade/dms-graphql-server
# [Feature] Setup apollo server

### What does this PR do?
- Replace the existing server implementation

### Description of tasks accomplished
- Replace the pre-existing server implementation
- Create typeDefs and resolvers

### How to test?
- Start the app
- Navigate to the address provided on the console
- Check the docs to run any query
